src/network-layer/default/RelayDefaultNetworkLayer.js

```diff
@@ -121,17 +121,22 @@ class RelayDefaultNetworkLayer {
         }
       }
       init = {
+        ...this._init,
         body: formData,
         method: 'POST',
       };
     } else {
       init = {
+        credentials: 'same-origin',
```

Comment:
I guess the better question would be: when **shouldn't** cookies be sent? :wink: